lib/transaction/enrollment_confirmation_step.js

```diff
@@ -44,7 +55,10 @@ enrollmentConfirmationStep.prototype.confirm = function confirm(data) {
   };
 
   var confirmTask = function confirmTask(done) {
-    self.strategy.confirm(data, done);
+    self.strategy.confirm(data, function confirmationDataAccepted(err, result) {
+      acceptedCallback(err, { recoveryCode: self.enrollmentAttempt.getRecoveryCode() });
```

Comment:
passing the recovery code here don't quite convince me, but, how are you going to get the recovery code otherwise... making `self.enrollmentAttempt.getRecoveryCode()` public seems like opening the door too much  considering that we are going to change how `enrollmentAttempt` works.